src/main/java/Duke.java

```diff
@@ -47,11 +47,9 @@ public static void main(String[] args) {
                     String arr2[] = second.split("/by",2);
```

Comment:
Array specifiers must be attached to the type not the variable.
should be String[] arr2